lib/still/compiler/incremental/node.ex

```diff
@@ -102,109 +71,58 @@ defmodule Still.Compiler.Incremental.Node do
   end
 
   @impl true
-  def handle_call(:compile, _from, %{cached_source_file: source_file} = state)
+  def handle_call({_, use_cache: true}, _from, %{cached_source_file: source_file} = state)
       when not is_nil(source_file) do
     {:reply, source_file, state}
   end
 
-  def handle_call(:compile, _from, state) do
-    case Compile.run(state) do
-      {:ok, source_file} ->
-        ErrorCache.set({:ok, source_file})
-        {:reply, source_file, %{state | cached_source_file: source_file}}
-
-      other ->
-        {:reply, other, state}
-    end
-  catch
-    _, %PreprocessorError{} = error ->
-      handle_compile_error(error)
+  def handle_call({:compile, _}, from, state) do
+    froms = all_waiting_compile([from])
```

Comment:
What's the need for this?

Reply:
It reduces the number of compiles. For instance, if 5 processes call the `compile` we compile once and reply to all of the same at once.

Reply:
Got it 👍